using System;

namespace ReversiLab.Exceptions
{
    internal class AlreadyStartedException : Exception
    {
    }
}